acts_as_archival.gemspec

```diff
@@ -33,6 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rr"
   gem.add_development_dependency "sqlite3"
+  gem.add_development_dependency "rubocop", "~> 0.47.1"
```

Comment:
I _think_ that the logic behind that is that newer versions of RuboCop add new cops, potentially making the build fail (at least if the PR is accepted with RuboCop being run on Travis before tests). At least that's why I lock it down in our own projects at work.

It's also mentioned in this section of the [RuboCop README](https://github.com/bbatsov/rubocop#installation).